src/pheval_exomiser/post_process/assess_prioritisation.py

```diff
@@ -0,0 +1,471 @@
+#!/usr/bin/python
+
+import csv
+import json
+import os
+from collections import defaultdict
+from dataclasses import dataclass, field
+from pathlib import Path
+from statistics import mean
+
+import click
+import pandas as pd
+
+from ..utils.file_utils import files_with_suffix
+from ..utils.phenopacket_utils import PhenopacketReader
+
+
+@dataclass
+class PrioritisationRanks:
```

Comment:
Isnt this code supposed to be general for all of pheval? Dont do it without making sure we all understand the same thing, but this seems like it should be in https://github.com/monarch-initiative/pheval/blob/main/src/pheval/cli_pheval_utils.py (or the corresponding package underneath that)

Reply:
Some of this is general pheval code, other bits are Exomiser-specific. These should be separated and the generic pheval code tested with another tool e.g. the XRare case.

Note that not all tools rank variants - some only rank genes so these cases need to be able to run too.

Reply:
> 



> Some of this is general pheval code, other bits are Exomiser-specific. These should be separated and the generic pheval code tested with another tool e.g. the XRare case.
> 
> Note that not all tools rank variants - some only rank genes so these cases need to be able to run too.

In this case should I also separate where the gene and variant rank methods are being called? By default it is running both from the click command but I can change this fairly easily

Reply:
Okay sure, I'll keep it in mind